### PR TITLE
Rename imports and cleanup goreleaser config

### PR DESCRIPTION
**Pull request by anweiss**
Addresses #36. CC @minhaj10p and @asadullah-yousuf-10p for review.

### CODE REVIEW

File: types/oscal/profile/profile.go [hunk 1]
```diff
@@ -33,15 +33,15 @@ type Merge struct {
 // A Custom element frames a structure for embedding represented controls in
 // resolution.
 type Custom struct {
-	IdSelectors      []Call  `xml:"call,omitempty" json:"calls,omitempty"`
```
Comment (minhaj10p) [inline types/oscal/profile/profile.go]: Is this reverted by intention? the profiles at hand have `calls` attributes and not `idSelectors`

File: types/oscal/profile/profile.go [hunk 2]
```diff
@@ -118,11 +118,6 @@ type Combine struct {
 // elements or attributes.
 type AsIs string
 
-func (a *AsIs) UnmarshalJSON(b []byte) error {
```
Comment (minhaj10p) [inline types/oscal/profile/profile.go]: this may cause errors in unmarshalling profiles

File: types/oscal/profile/profile.go [hunk 3]
```diff
@@ -55,20 +55,20 @@ type Modify struct {
...
 type Exclude struct {
-	IdSelectors      []Call  `xml:"call,omitempty" json:"calls,omitempty"`
+	IdSelectors      []Call  `xml:"call,omitempty" json:"idSelectors,omitempty"`
 	PatternSelectors []Match `xml:"match,omitempty" json:"patternSelectors,omitempty"`
 }
 
 // A parameter setting, to be propagated to points of insertion
 type SetParam struct {
-	Id           string               `xml:"param-id,attr,omitempty" json:"id,omitempty"`
+	Id           string               `xml:"id,attr,omitempty" json:"id,omitempty"`
```
Comment (minhaj10p) [inline types/oscal/profile/profile.go]: I believe we had a discussion on this and you created a ticket for the upstream. Is this change by intention?

**Comment by anweiss** [comment]
@minhaj10p the profile types were re-generated from the metaschemas via https://github.com/docker/oscalkit/blob/master/Makefile#L24. This process is managed in https://github.com/docker/oscalkit/tree/master/metaschema and ultimately comes from the OSCAL profile metaschema at https://github.com/usnistgov/OSCAL/blob/master/schema/metaschema/oscal-profile-metaschema.xml. As such, we need to fix this in our metaschema generation process.

File: types/oscal/profile/profile.go (hunk 1 above)
Comment (anweiss) [reply types/oscal/profile/profile.go]: This was re-generated from the profile metaschema which, as we've discussed, has a bug per https://github.com/usnistgov/OSCAL/issues/287. I'll revert these changes.

File: types/oscal/profile/profile.go (hunk 2 above)
Comment (anweiss) [reply types/oscal/profile/profile.go]: This was re-generated from the profile metaschema which, as we've discussed, has a bug per https://github.com/usnistgov/OSCAL/issues/287. I'll revert these changes.

File: types/oscal/profile/profile.go (hunk 3 above)
Comment (anweiss) [reply types/oscal/profile/profile.go]: This was re-generated from the profile metaschema which, as we've discussed, has a bug per https://github.com/usnistgov/OSCAL/issues/287. I'll revert these changes.

Comment (minhaj10p) [reply types/oscal/profile/profile.go]: awesome. other than this everything LGTM, Please go ahead and merge when you find time with these changes and i'll rebase the dependent PR

Comment (anweiss) [reply types/oscal/profile/profile.go]: Will do. Thanks

**Comment by anweiss** [comment]
@minhaj10p please submit an Approval review so I can merge. Thanks

**Pull request merged**